src/sherpa_ai/connectors/chroma_vector_store.py

```diff
@@ -0,0 +1,78 @@
+import uuid
+
+import chromadb
+from chromadb.utils import embedding_functions
+from langchain.docstore.document import Document
+from langchain.text_splitter import CharacterTextSplitter
+
+import sherpa_ai.config as cfg
+
+
+class ChromaVectorStore:
+    def __init__(self, db) -> None:
+        self.db = db
+
+    @classmethod
+    def chroma_from_texts(cls, texts, embedding=None, meta_datas=None):
+        openai_ef = embedding_functions.OpenAIEmbeddingFunction(
```

Comment:
Make this either as a parameter to the function and use OpenAIEmbeddingFunction as the default if the parameter is None